### ISSUE TITLE
Cannot reproduce RACE score 

### ISSUE THREAD

**Issue by sugiyama-hiroaki** [openai/finetune-transformer-lm#11]
Hi, 

We tried several settings based on your code and paper, but unfortunately we cannot reproduce the RACE score (the training loss decreases, but the dev accuracy only reaches 0.26). 
Could you tell me some tips about the parameter or code modification for achieving the performance?

Thanks

**Comment by hardik2396** [comment]
Any updates?  I also trying to reproduce RACE results.

**Comment by sugiyama-hiroaki** [comment]
It works! I had a simple bug on converting problems to tensor format.

**Comment by hardik2396** [comment]
How much accuracy did you get on RACE? 
I am getting 53.

**Comment by sugiyama-hiroaki** [comment]
I had 57.2. This is slightly lower than the paper score, but I think it's a reasonable level.

**Comment by hardik2396** [comment]
I am trying but due to the low number of GPU, I cannot reproduce the results. I would be very grateful to you if you can provide me pre-trained weights.  @sugiyama-hiroaki

**Comment by hardik2396** [comment]
Can you share the code for RACE ? @sugiyama-hiroaki

**Comment by sugiyama-hiroaki** [comment]
I'm so sorry but I cannot share my weights because I rewrote the code for adapting to RACE.
Besides, I'm a company researcher so I don't have a permission to share my code.

If you don't have enough number of GPUs, you can train the model with skipping the update function until it reaches the batch size.

**Comment by chengchingwen** [comment]
@sugiyama-hiroaki Thanks for the tips!
I'm also trying to reimplement for adapting to RACE, but I can't get higher than 35% acc on dev & test set and 58% acc on training set. Could you please tell me what hyperparameters did you choose for training the model?

**Comment by ap229997** [comment]
@sugiyama-hiroaki Can you tell whether the test set scores were fluctuating a lot or stable (like uniformly going up or a smooth pattern) with respect to the epochs. Also can you tell the number of epochs you trained the model for?
